docs/admin/CertificateTransparency.adoc

```diff
@@ -1,8 +1,10 @@
 = Certificate Transparency
+Christina Fu <cfu@redhat.com>
+:toc:
 
 == Overview
 
-A basic version of Certificate Transparency (CT) V1 support (rfc 6962) is offered in this release. It has the capability of issuing certificates with embedded Signed Certificate Time stamps (SCTs) from any trusted log where each deployment site chooses to have its root CA cert included. The system can also be configured to support multiple CT logs.  Minimum of one trusted CT log is required for this feature to work.
+A basic version of Certificate Transparency (CT) V1 support (rfc 6962) is offered since PKI 10.6. It has the capability of issuing certificates with embedded Signed Certificate Time stamps (SCTs) from any trusted log where each deployment site chooses to have its root CA cert included. The system can also be configured to support multiple CT logs.  Minimum of one trusted CT log is required for this feature to work.
```

Comment:
Shouldn't this be PKI 10.9?

Reply:
ok.  For some reason I found reference of 10.6 somewhere.  I just found a difference of 10.9 so I'll go with that. thanks!

Reply:
fixed. thanks.